src/Processors/QtiV2/In/RubricBlockMapper.php

```diff
@@ -72,20 +81,7 @@ public function parseWithRubricBlockComponent(RubricBlock $rubricBlock, $foundSc
                     $mapper = new DistractorRationaleResponseMapper();
                     $distractorRationalResponseLevelArray = $mapper->parseWithDistractorRationaleResponseComponent($rubricBlock);
                     $result ['question_metadata'] = $distractorRationalResponseLevelArray;
-                }
-                break;
-
-            case ($rubricBlock->getUse() === 'stimulus'):
-                if ($views->contains(View::CANDIDATE)) {
-                    $contents = QtiMarshallerUtil::marshallCollection($rubricBlock->getContent());
-                    $result = [
-                        'stimulus' => $contents,
-                    ];
-                }
-                break;
-
-            case ($rubricBlock->getClass() === 'DistractorRationale'):
```

Comment:
We still need to support this key. It should be a fall-through case so we can handle both the same way, e.g.:

```
case ($rubricBlock->getUse() === 'rationale'):
    /* falls through */
case ($rubricBlock->getClass() === 'DistractorRationale'):
```

Reply:
As per previous comment this is differentiated by views whether it is distractor rationale response level or only distractor rationale . 
Means use is rationale but views is candidate it would be for student and view is author than it would be for teacher .

Reply:
@rakeshqdsa I'm not sure I follow. I know that we support two types of views separately for this scenario here. I'm talking about also handling both `use="rationale"` and `class="DistractorRationale"` cases. It is valid for QTI content to use either to achieve the same purpose. If we remove support for `class="DistractorRationale"`, then we break support for existing QTI content. Both of these cases need to behave the same way, hence my suggestion to include both cases.

Reply:
Done